tests/attr/test_feature_ablation.py

```diff
@@ -183,7 +183,7 @@ def test_error_ablations_per_eval_limit_batch_scalar(self):
         inp = torch.tensor([[2.0, 10.0, 3.0], [20.0, 50.0, 30.0]], requires_grad=True)
         ablation = FeatureAblation(lambda inp: torch.sum(net(inp)).item())
         with self.assertRaises(AssertionError):
-            _ = ablation.attribute(inp, ablations_per_eval=2,)
+            _ = ablation.attribute(inp, ablations_per_eval=2)
```

Comment:
Had to include this lint change as well as the test_gradient_shap change, since the conda test on CircleCI was not passing without it